Sources/KSCrashInstallations/KSCrashInstallation.m

```diff
@@ -317,6 +319,9 @@ - (void)sendAllReportsWithCompletion:(KSCrashReportFilterCompletion)onCompletion
         self.prependedFilters,
         sink,
     ] mutableCopy];
+    if (self.isDoctorEnabled) {
```

Comment:
The order seems unclear. Maybe it's better to create a mutable array, optionally add these filters with just `addObject:` and then add `self.prependedFilters` and `sink`?